packages/uows-client-generator/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "@stfc-user-programme/uows_client_generator",
+  "version": "1.0.13",
+  "description": "Generates an interface for consumning the User Office Web Service",
+  "main": "lib/index/index.js",
+  "author": "STFC",
+  "license": "ISC",
+  "types": "lib/index/index.d.ts",
+  "directories": {
+    "lib": "lib"
+  },
+  "files": [
+    "lib"
+  ],
+  "scripts": {
+    "test": "npm run build && jest --config jestconfig.json",
+    "clean": "rm -rf ./lib",
+    "build": "npm run clean && tsc -p ./tsconfig.build.json",
+    "start": "ts-node src/index/index.ts"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/UserOfficeProject/user-office-lib.git"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "dependencies": {
+    "@types/jest": "^26.0.15",
+    "@types/rewire": "^2.5.28",
+    "@types/soap": "^0.21.0",
+    "rewire": "^5.0.0",
+    "soap": "^0.34.0",
+    "ts-jest": "^26.4.4",
```

Comment:
Do we need this and `@types/jest` as runtime dependencies? I think we may be able to only install them as dev dependencies (see https://kulshekhar.github.io/ts-jest/user/install#dependencies; it seems that they're only adding it to the dev list)

Reply:
I've moved the packages used for testing into dev dependencies.